examples/query/canisters/rust/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "rust"
+version = "0.0.0"
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+candid = "0.7.14"
```

Comment:
Do you need the `candid` and `ic-cdk` dependencies here? Let's see if we can remove them

Reply:
When I got rid of ic-cdk I got this error
```
--> canisters/rust/src/lib.rs:1:1
  |
1 | #[ic_cdk_macros::query]
  | ^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `ic_cdk`
  |
  = note: this error originates in the attribute macro `ic_cdk_macros::query` (in Nightly builds, run with -Z macro-backtrace for more info)
```